plugins/tech-stack/src/plugin.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createPlugin, createRouteRef } from '@backstage/core';
+import ToolsComponent from './components/ToolsComponent';
+
+export const rootRouteRef = createRouteRef({
+  path: '/tech-stack',
+  title: 'tech-stack',
+});
+
+export const plugin = createPlugin({
+  id: 'tech-stack',
+  register({ router }) {
+    router.addRoute(rootRouteRef, ToolsComponent);
```

Comment:
Note this is now a deprecated method of adding routes.  Instead,

https://backstage.io/docs/plugins/plugin-development#routing

You'll find many examples inside the project now using the new pattern.